keras_cv/models/classification/image_classifier_test.py

```diff
@@ -109,6 +109,43 @@ class ImageClassifierPresetSmokeTest(tf.test.TestCase, parameterized.TestCase):
     def setUp(self):
         self.input_batch = tf.ones(shape=(2, 224, 224, 3))
 
+    @parameterized.named_parameters(
```

Comment:
Unless there's a good reason, we don't need to test the output of each possible preset. Just one should be enough to catch changes to the forward pass. I would drop

Reply:
Keeping until we LGTM, just to confirm that we don't accidentally shift architecture in code review.

Reply:
OK, remember to remove.

Reply:
It's still here tho?

Reply:
I kept one `b` variant and one normal variant.  They're slightly logically distinct so I intentionally kept one of each.  Before there were 4.

Reply:
actually - just the B variant should cover everything